### PR TITLE
README.md and SAMPLE.md

### PR DESCRIPTION
**Pull request by danielegiallonardo**
Please review the documentation contained in the README.md and SAMPLE.mc files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,40 +8,141 @@
...
+                o.DefaultSignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                o.DefaultChallengeScheme = SpidCieConst.AuthenticationScheme;
+            })
+            .AddCookie()
+            .AddSpidCieOIDC(o =>
+            {
+                o.LoadFromConfiguration(Configuration);
+
+                o.RequestRefreshToken = true;
+                o.SpidOPs.Add("http://127.0.0.1:8000/oidc/op/"); // Example SPID OP Url
```
Comment (peppelinux) [inline README.md:58]: For a good dockerimage strategy, is it possibile to retrieve these values from ENV or do we have in the documentation a way to dynamically configure these?

Comment (peppelinux) [reply README.md:58]: Does these parameter be compiled in C# or is this a configuration file that we can copy in a docker volume before starting the container?

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielegiallonardo) [reply README.md:58]: It's possible to read them from an external settings file, I'll add the feature

**Review by danielegiallonardo — COMMENTED** [review]
Settings now are loaded from an external configuration file

**Pull request merged**